# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../avifile/head-n1-usage.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- avifile-0.7-0.7.41/configure.in	2004-10-01 15:17:36.000000000 +0200
+++ avifile-0.7-0.7.41-t2/configure.in	2004-10-21 22:24:04.066025928 +0200
@@ -913,7 +913,7 @@
 WIN32_CFLAGS=`echo $CFLAGS | sed s/-fomit-frame-pointer//g`
 AC_SUBST(WIN32_CFLAGS)
 
-lt_major_version=`libtool --version 2>&1 | head -1 | sed 's/\(.*\ \)\([[0-9]]*\.[[0-9]]*\.[[0-9]]*\)\(.*\)/\2/'`
+lt_major_version=`libtool --version 2>&1 | head -n1 | sed 's/\(.*\ \)\([[0-9]]*\.[[0-9]]*\.[[0-9]]*\)\(.*\)/\2/'`
 lt_minor_version=`echo $lt_major_version | cut -d . -f 2`
 lt_micro_version=`echo $lt_major_version | cut -d . -f 3`
 lt_major_version=`echo $lt_major_version | cut -d . -f 1`
